### PR TITLE
Enable extraction of some data from efolder documents

### PR DESCRIPTION
**Pull request by brianbegy**
Extracts the attachments and saves what data we have to a new element in metadata.

### CODE REVIEW

File: GuaranteedRate.Sextant.ConsoleTester/App.config [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+    </startup>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-7.0.0.0" newVersion="7.0.0.0" />
```
Comment (jeffpsherman) [inline GuaranteedRate.Sextant.ConsoleTester/App.config]: This seems wrong - we're using json 10

Comment (brianbegy) [reply GuaranteedRate.Sextant.ConsoleTester/App.config]: Yeah.  Why isn't Nuget doing the redirects correctly?

File: GuaranteedRate.Sextant/EncompassUtils/FieldUtils.cs [hunk 2]
```diff
@@ -56,6 +57,9 @@ public class FieldUtils
         private const string MORTGAGES_STARTS = "FM";
         private const string VESTING_PARITIES_STARTS = "TR";
 
+        private static readonly string[] END_INDEX_EXCLUDE_LIST = {"IRS4506.X61.01"};
```
Comment (jeffpsherman) [inline GuaranteedRate.Sextant/EncompassUtils/FieldUtils.cs]: I also see `IRS4506.X62`

File: GuaranteedRate.Sextant/EncompassUtils/FieldUtils.cs [hunk 3]
```diff
@@ -56,6 +57,9 @@ public class FieldUtils
         private const string MORTGAGES_STARTS = "FM";
         private const string VESTING_PARITIES_STARTS = "TR";
 
+        private static readonly string[] END_INDEX_EXCLUDE_LIST = {"IRS4506.X61.01"};
+        private static readonly string[] MIDDLE_INDEX_EXCLUDE_LIST = { "AUSTRACKING.AUS.X10" };
```
Comment (jeffpsherman) [inline GuaranteedRate.Sextant/EncompassUtils/FieldUtils.cs]: Logging also shows `AUSTRACKING.AUS.X100`

Comment (jeffpsherman) [reply GuaranteedRate.Sextant/EncompassUtils/FieldUtils.cs]: Actually, the stalker config is
```
Stalker.IgnoreFields=IRS4506.X61,IRS4506.X62,AUSTRACKING.AUS.X100
```

So you probably left off  a 0

**Pull request merged**